docs/src/pages/demos/buttons/IconLabelButtons.js

```diff
@@ -34,7 +34,8 @@ function IconLabelButtons(props) {
       </Button>
       <Button variant="contained" color="primary" className={classes.button}>
         Send
-        <SendIcon className={classes.rightIcon}>send</SendIcon>
+        {/* Uses Font Icons, see the installation instructions in the docs. */}
```

Comment:
{/* This Button uses a Font Icon, see the installation instructions in the docs. */}